src/controllers/blocks/BlocksController.ts

```diff
@@ -114,9 +114,11 @@ export default class BlocksController extends AbstractController<BlocksService>
 		const eventDocsArg = eventDocs === 'true';
 		const extrinsicDocsArg = extrinsicDocs === 'true';
 
-		const paramFinalized = finalized === 'true' ? true : false;
 		const { hash, queryFinalizedHead, omitFinalizedTag } =
-			await this.parseFinalizationOpts(this.options.finalizes, paramFinalized);
+			await this.parseFinalizationOpts(
```

Comment:
I don't get this, how does it fix it by converting passing it as a `String` rather than a `boolean`?

Reply:
hmmm, you make a good point that it is confusing too outside eyes as to what is happening, so I am going to just revert it completely to what it use to be so that there is no confusion.

Reply:
I'm a bit confused about the string usage; can the string just be `true` or `false` and, why a string and not a bool? I'm probably missing too much context really :)

Sounds like you'll revert anyway so maybe I don't need to know :D

Reply:
@niklasad1 @jsdw Okay, so I reverted it back to what it was originally before I abstracted out of the controller. Hopefully it's a little clearer